src/command.ts

```diff
@@ -126,37 +119,7 @@ export abstract class Command {
 
   public static hasDynamicHelp = false
 
-  protected static '_--' = false
-
-  protected static _enableJsonFlag = false
-
-  public static get enableJsonFlag(): boolean {
-    return this._enableJsonFlag
-  }
-
-  public static set enableJsonFlag(value: boolean) {
-    this._enableJsonFlag = value
-    if (value === true) {
-      this.baseFlags = jsonFlag
-    } else {
-      delete this.baseFlags?.json
-      this.flags = {} // force the flags setter to run
-      delete this.flags?.json
-    }
-  }
-
-  public static get '--'(): boolean {
-    return Command['_--']
-  }
-
-  public static set '--'(value: boolean) {
-    Command['_--'] = value
-  }
-
-  public get passThroughEnabled(): boolean {
-    return Command['_--']
-  }
-
+  public static enableJsonFlag = false
```

Comment:
this whole file is a LOT better.  I know all the complexity moved to the prototype crawler, but that's good for consumers.